app/generator/content/part3.js

```diff
@@ -31,8 +31,7 @@ const part3 = () => {
           'Find other ways to get in touch at ',
           { text: 'www.gov.uk/guidance/contacting-the-rpa-about-sfi', link: 'https://www.gov.uk/guidance/contacting-the-rpa-about-sfi', style: 'link' }
         ]
-      },
-      { text: 'To receive this statement in large print or another alternative format, contact the Rural Payments Agency.', style: 'header2' }
```

Comment:
Assume this has been scraped by UR? Am I right in saying this is still very much to be signed-off context-wise?

Reply:
Apparently so.  Silently dropped.

AFAIK only unknown wording left is the contact RPA text.